src/sources/hawker/index.ts

```diff
@@ -0,0 +1,102 @@
+import { Hawker, HawkerRaw } from './model';
+
+import autoParse from '../../util/autoParse-datagov';
+
+const splitDate = (dateString: string) => {
```

Comment:
I'd suggest importing moment instead of using a special utility function (there may be no guarantee on the returned date format from the API)

Reply:
is there any reason this function is still required? seems like it only serves as a check now that Moment is being used